mod/messageboard/start.php

```diff
@@ -114,7 +114,11 @@ function messageboard_add($poster, $owner, $message, $access_id = ACCESS_PUBLIC)
 			$poster->getURL()
 		), $owner->language);
 
-		notify_user($owner->guid, $poster->guid, $subject, $body);
+		$params = [
+			'action' => 'create',
```

Comment:
The notifications system has been designed to take the action name (`create`, `delete`, `reset`, `mention`, etc) but not the whole action path (e.g. `messageboard/add`).

When using `create` as the action, we already get automatically e.g:
 - Translation string: `notification:create:object:messageboard:subject`
 - Send hook: `send, notification:create:object:messageboard`

While it is true, that the examples above are generated only by the asynchronous notification system (and this PR is adding them to synchronous notifications) I would like to keep the action names consistent.